src/main/java/com/faforever/client/replay/LoadLocalReplaysTask.java

```diff
@@ -26,7 +26,7 @@ public LoadLocalReplaysTask(ReplayService replayService, I18n i18n) {
   @Override
   protected Collection<Replay> call() throws Exception {
     updateTitle(i18n.get("replays.loadingLocalTask.title"));
-    return replayService.getLocalReplays();
+    return replayService.getLocalReplays().get();
```

Comment:
Now get just shifted further uppwards

Reply:
There has to be a solution for this, let's review it together

Reply:
ok lets do that